### PR TITLE
Register cat.thedev.is-a.dev

### PR DESCRIPTION
**Pull request by Dogeyisyes**
Register catdev.is-a.dev with TXT record pointing to https://github.com/Dogeyisyes/2egf/blob/main/asd.

### CODE REVIEW

File: domains/cat.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+        "owner": {
+           "username": "Dogeyisyes",
+           "email": "dogegamesroblox@gmail.com",
+           "discord": "1128124810619273326"
+        },
+    
+        "record": {
+            "CNAME": "https://github.com/dogeyisyes/asdf3/blob/main/yes"
```
Comment (DEV-DIBSTER) [inline domains/cat.json:9]: Invalid CNAME record. CNAME records must be single string hostname's without any protocols or path URLs attached.

Comment (DEV-DIBSTER) [reply domains/cat.json:9]: Also this page returns a 404.

File: domains/cat.thedev.json [hunk 2]
```diff
@@ -0,0 +1,12 @@
+{
+        "owner": {
+           "username": "Dogeyisyes",
+           "email": "dogegamesroblox@gmail.com",
+           "discord": "1128124810619273326"
+        },
+    
+        "record": {
+            "TXT": "https://github.com/dogeyisyes/2egf/blob/main/asd"
```
Comment (DEV-DIBSTER) [inline domains/cat.thedev.json:9]: TXT records alone doesn't do anything. Link leads to a 404 page. Invalid records.

**Review by DEV-DIBSTER — CHANGES_REQUESTED** [review]

File: domains/cat.thedev.json
Comment (CuteDog5695) [inline domains/cat.thedev.json:1]: thedev.is-a.dev does not exist, so you can't make a nested subdomain with it

**Review by iostpa — CHANGES_REQUESTED** [review]

**Pull request closed without merging**